README.md

```diff
@@ -1,3 +1,9 @@
+# yaml
```

Comment:
Don't include the package name here – redundant!

Reply:
It's a common practice for packages as far as I'm aware (https://github.com/dart-lang/dart-pad and others).

Reply:
A common practice we should undo. It's completely redundant - see https://pub.dartlang.org/packages/angular_forms

Reply:
*something something hobgoblins something something*

Reply:
I down-vote your opinion, but will update this PR.